core/domain/base_model_validators.py

```diff
@@ -60,11 +60,14 @@
 ERROR_CATEGORY_TIME_FIELD_CHECK = 'time field relation check'
 ERROR_CATEGORY_TYPE_CHECK = 'type check'
 ERROR_CATEGORY_VERSION_CHECK = 'version check'
+ERROR_CATEGORY_STALE_CHECK = 'stale check'
 
 VALIDATION_MODE_NEUTRAL = 'neutral'
 VALIDATION_MODE_STRICT = 'strict'
 VALIDATION_MODE_NON_STRICT = 'non-strict'
 
+PERIOD_TO_HARD_DELETE_MODELS_MARKED_AS_DELETED = datetime.timedelta(weeks=8)
```

Comment:
Any specific reason for defining this constant in multiple places?

Reply:
But we are defining constant so that it can be changed easily in the future if required. If we define this constant in two places then there is a high chance of missing to update the other value while updating one. (We rate expecting these duplicate constant to have the same value, right?)

Reply:
Yeah, I guess I'm quite indifferent about it. Done.